src_python/habitat_sim/utils/settings.py

```diff
@@ -13,12 +13,12 @@
     "default_agent": 0,
     "sensor_height": 1.5,
     "hfov": 90,
-    "color_sensor": True,  # RGB sensor (default: ON)
-    "semantic_sensor": False,  # semantic sensor (default: OFF)
-    "depth_sensor": False,  # depth sensor (default: OFF)
-    "ortho_rgba_sensor": False,  # Orthographic RGB sensor (default: OFF)
-    "ortho_depth_sensor": False,  # Orthographic depth sensor (default: OFF)
-    "ortho_semantic_sensor": False,  # Orthographic semantic sensor (default: OFF)
+    "color_sensor": True,
```

Comment:
Comments should be repaired / replaced?

Reply:
Karmesh suggested I remove them bc they are kind of redundant.  I can replace them though if anyone finds them useful